coeus-impl/src/main/resources/ApplicationResources.properties

```diff
@@ -826,9 +827,9 @@ error.award.awardhierarchy.sync.invalidstatus=The Award {0} must have an active
 error.award.awardhierarchy.sync.locked=This Award has been locked for an Award Hierarchy Sync operation by document number {0}. 
 
 #START - Overriden rice properties
-app.feedback.linkText=Provide Feedback
+app.feedback.linkText=Get Help
 #To not show a feedback link, simply leave the feedback link blank.
-app.feedback.link=https://test.kuali.org/jira/secure/CreateIssue.jspa?pid=10310&issuetype=1
+app.feedback.link=mailto:support@rsmart.com?Subject=Help
```

Comment:
Well I think this is correct right now for CX customers. We should update this to KualiCo, but do we have a support email? We should also get this out of the resources, but isn't this a Rice thing? Maybe after we have our fork of Rice finalized we can fix this there?